tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/jdbc/RelationalDataSource.java

```diff
@@ -18,7 +18,7 @@
 import java.util.Iterator;
 import java.util.List;
 
-public interface JdbcTableDataSource
+public interface RelationalDataSource
```

Comment:
This is somehow wierd. `HiveTableDefinition` can be also though as `relational`, in that sense this name is wrong.

If cassandra does not support JDBC maybe we would need some special `CassandraTableDefinition` as we have for hive. It think that way seems more natural.